cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/network/NetworkResourcesCreationRequest.java

```diff
@@ -25,6 +27,8 @@
 
     private final boolean privateEndpointsEnabled;
 
+    private final String existingPrivateDnsZoneId;
```

Comment:
This and the privateEndpointsEnabled come from the base network however they Azure specific parameters and should be part of the [BaseNetwork's AzureParams object](https://github.com/hortonworks/cloudbreak/blob/2.54.0-b37/structuredevent-model/src/main/java/com/sequenceiq/environment/network/dto/NetworkDto.java#L32)